README.md

```diff
@@ -10,6 +10,9 @@ The ODF CLI tool provides configuration and troubleshooting commands for OpenShi
   More information about the ceph subsystems can be found [here](https://docs.ceph.com/en/latest/rados/troubleshooting/log-and-debug/#ceph-subsystems)
 - `odf ceph purge-osd <ID>`: Permanently remove an OSD from the cluster.
 - `odf help` : Display help text
+- `odf subvolume ls`: Display all the subvolumes
+- `odf subvolume ls --stale`: Display all the stale subvolumes that can be deleted.
```

Comment:
Let's not include all options in this main readme, just the core commands. They can see the sub help topic for details.
```suggestion
```